### ISSUE TITLE
Xaminals.Android sample - routing issue

### ISSUE THREAD

**Issue by artyom-p** [xamarin/MobileBlazorBindings#101]
Hi, im unable to launch Xaminals sample, while in debug it shows the following exception:

"No Content found for ShellContent, Title:About, Route about"

`  at Xamarin.Forms.ShellContent.Xamarin.Forms.IShellContentController.GetOrCreateContent () [0x00046] in d:\a\1\s\Xamarin.Forms.Core\Shell\ShellContent.cs:70 
  at Xamarin.Forms.Platform.Android.ShellSectionRenderer.OnCreateView (Android.Views.LayoutInflater inflater, Android.Views.ViewGroup container, Android.OS.Bundle savedInstanceState) [0x000d2] in d:\a\1\s\Xamarin.Forms.Platform.Android\Renderers\ShellSectionRenderer.cs:165 
  at Android.Support.V4.App.Fragment.n_OnCreateView_Landroid_view_LayoutInflater_Landroid_view_ViewGroup_Landroid_os_Bundle_ (System.IntPtr jnienv, System.IntPtr native__this, System.IntPtr native_inflater, System.IntPtr native_container, System.IntPtr native_savedInstanceState) [0x00020] in <fefee6c2c695459088a9df092723e052>:0 
  at (wrapper dynamic-method) Android.Runtime.DynamicMethodNameCounter.54(intptr,intptr,intptr,intptr,intptr)`

dotnet --info:

Host (useful for support):
  Version: 3.1.2
  Commit:  916b5cba26

.NET Core SDKs installed:
  2.1.801 [C:\Program Files\dotnet\sdk]
  2.2.301 [C:\Program Files\dotnet\sdk]
  2.2.402 [C:\Program Files\dotnet\sdk]
  3.1.200-preview-015002 [C:\Program Files\dotnet\sdk]
  3.1.200 [C:\Program Files\dotnet\sdk]

.NET Core runtimes installed:
  Microsoft.AspNetCore.All 2.1.12 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.All]
  Microsoft.AspNetCore.All 2.1.16 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.All]
  Microsoft.AspNetCore.All 2.2.5 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.All]
  Microsoft.AspNetCore.All 2.2.6 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.All]
  Microsoft.AspNetCore.All 2.2.7 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.All]
  Microsoft.AspNetCore.App 2.1.12 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 2.1.16 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 2.2.5 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 2.2.6 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 2.2.7 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 3.1.2 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 2.1.12 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.1.16 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.2.5 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.2.6 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 2.2.7 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 3.1.2 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.WindowsDesktop.App 3.1.2 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]

**Comment by Eilon** [comment]
Hi @artyom-p sorry you ran into an issue here. Were you running the sample from cloning this repo from the latest source code?

**Comment by artyom-p** [comment]
Yes, re-cloned the repo again, just a moments ago, the same issue.

**Comment by Eilon** [comment]
Thank you that is helpful. I will try to reproduce the issue and then investigate.

**Comment by trippedoutfish** [comment]
Hi @Eilon I am having the same issue. 

I believe it has something to do with the <Tab> item spanning line 14-28 of AppShell.razor as commenting it out (but not the ShellContents inside of it) causes it to work for me.

Let me know if there is some way in which I can help you solve this issue.

**Comment by Eilon** [comment]
OK I can confirm that this repros for me too. I think I saw this recently while doing a major refactoring but then forgot to investigate it again. Thank you for all the details, I agree there is definitely a bug here.
